lib/mutant/repository.rb

```diff
@@ -20,17 +20,9 @@ def call(subject)
 
     # Diff between two objects in repository
     class Diff
-      include Adamantium, Concord.new(:from, :to)
+      include Adamantium, Anima.new(:config, :from, :to)
```

Comment:
Good point. With things like `from` and `to` as positional arguments sometimes the order would be different depending on the "direction" you are modelling. Having them as Hash arguments make sense.

Although I could argue that `config`, assuming it is required, might be better as the first required argument to the constructor.